modules/proc_configuring-by-using-the-ocp-web-console.adoc

```diff
@@ -0,0 +1,32 @@
+:_mod-docs-content-type: PROCEDURE
+[id="configuring-by-using-the-ocp-web-console_{context}"]
+= Configuring NotificationsConfiguration (CR) by using the {OCP} web console
+
+You can configure `NotificationsConfiguration` (CR) by using the {OCP} web console.
+
+.Prerequisites
+
+* You have access to an {OCP} cluster with `cluster-admin` privileges and are logged into the web console.
+* You have installed the {gitops-title} Operator on your cluster.
+* You have enabled notifications for the Argo CD instance. For more information, see *Enabling notifications with an Argo CD instance*.
+
+.Procedure
+
+. Go to the *NotificationsConfiguration* page and click `default-notifications-configuration`.
+. On the *default-notifications-configuration* page, click *YAML* and add the configuration for any supported resources such as Templates, Triggers, Services, and Subscriptions. For example, under `Templates` in the code, add the following sample configuration for *Templates*:
++
+.Example
+[source,yaml]
+----
+  template.my-custom-template: |
+    message: |
+    Application details: {{.context.argocdUrl}}/applications/{{.app.metadata.name}}.
```

Comment:
Again, we could possibly add some callouts. Here it would be anough to just add a callout that would say: this configuration adds a custom template to the the  `NotificationsConfiguration` CR. 

or something similar.

Reply:
Fixed.